extra/readline/README

```diff
@@ -76,65 +76,62 @@ instance, FreeBSD 4.2 with any version of gcc is identified as
 In the stanza for your operating system-compiler pair, you will need to
 define several variables.  They are:
 
-SHOBJ_CC	The C compiler used to compile source files into shareable
-		object files.  This is normally set to the value of ${CC}
-		by configure, and should not need to be changed.
+SHOBJ_CC    The C compiler used to compile source files into shareable
+        object files.  This is normally set to the value of ${CC}
+        by configure, and should not need to be changed.
 
-SHOBJ_CFLAGS	Flags to pass to the C compiler ($SHOBJ_CC) to create
-		position-independent code.  If you are using gcc, this
-		should probably be set to `-fpic'.
+SHOBJ_CFLAGS    Flags to pass to the C compiler ($SHOBJ_CC) to create
+        position-independent code.  If you are using gcc, this
+        should probably be set to `-fpic'.
 
-SHOBJ_LD	The link editor to be used to create the shared library from
-		the object files created by $SHOBJ_CC.  If you are using
-		gcc, a value of `gcc' will probably work.
+SHOBJ_LD    The link editor to be used to create the shared library from
+        the object files created by $SHOBJ_CC.  If you are using
+        gcc, a value of `gcc' will probably work.
 
-SHOBJ_LDFLAGS	Flags to pass to SHOBJ_LD to enable shared object creation.
-		If you are using gcc, `-shared' may be all that is necessary.
-		These should be the flags needed for generic shared object
-		creation.
+SHOBJ_LDFLAGS    Flags to pass to SHOBJ_LD to enable shared object creation.
+        If you are using gcc, `-shared' may be all that is necessary.
+        These should be the flags needed for generic shared object
+        creation.
 
-SHLIB_XLDFLAGS	Additional flags to pass to SHOBJ_LD for shared library
-		creation.  Many systems use the -R option to the link
-		editor to embed a path within the library for run-time
-		library searches.  A reasonable value for such systems would
-		be `-R$(libdir)'.
+SHLIB_XLDFLAGS    Additional flags to pass to SHOBJ_LD for shared library
+        creation.  Many systems use the -R option to the link-editor to embed a path within the library for run-time
```

Comment:
For readability, please keep to the line-length convention rather than introducing long run-on lines